### ISSUE TITLE
Suggestion - New Ideas for PD

### ISSUE THREAD

**Issue by Mugdha57** [InZidiuZ/op-framework-issue-tracker#3550]
### Assurance

I have read the `README.md`

### Summary

1. Increase Storage Capacity for Police

Currently, all players have 250 storage units in their pockets, but those who do PD roleplay have to carry a lot of items, for example- https://files.catbox.moe/vtt6a0.png | My suggestion is to add 300 storage units (SU) for PD

2. Add Ability for Passengers to Control Police Lights and Sirens 

If the vehicle is a marked police car, the passenger should have full control over lights and sirens if sitting in the front seat. This will add realism to the PD RP experience, 

### Reason

1. When we arrest someone, it’s difficult to confiscate items with only 250 storage units.

2. Recently, passengers have been able to turn the PD radar on and off from the passenger seat, so it would be great if they could also control the lights and sirens.

**Comment by SpiderCord** [comment]
Yes its needed... Lots of item have to carry with a PD character...and when its time to collect some evidence its verry annoying, drop something of pockets the take evidence. Please do this. Its help us alot.

**Comment by TheIram** [comment]
Sometimes while taking evidence some officers drop their vests for not having enough storage. PD need it for more quality-full RP...

**Comment by Professional3oT** [comment]
Or else can add  extra bag which can only give your pd dress a new smart look just like a pouch on the leg can be use as a extra storage

**Comment by Psycho1952** [comment]
Yes i agree with this PD needs more space i either have to armor or drop armor to take guns possesions of the suspects we are almost always full with our space but now i have to decide if i want to have food on me or no or less armors so i can seize the possesions of the person what has 2 guns 5 oxy a ligther gas can etc etc its annoying....
